sandbox/src/test/java/org/robolectric/internal/bytecode/ClassInstrumentorTest.java

```diff
@@ -39,6 +41,26 @@ protected byte[] getClassBytes(String className) {
         };
   }
 
+  @Test
+  public void instrumentMethod_legacy() {
```

Comment:
nit: rename this `instrumentRegularMethod`. The `legacy` doesn't apply here, the regular method instrumentation is not legacy. It's just the difference between instrumenting a `native` method (i.e. backed by JNI native code) or instrumenting a regular Java method.